packages/docusaurus-1.x/lib/copy-examples.js

```diff
@@ -140,34 +135,28 @@ if (feature === 'translations') {
 } else {
   const folder = path.join(__dirname, '..', 'examples', 'basics');
   // copy docs examples
-  if (fs.existsSync(`${CWD}/../docs-examples-from-docusaurus`)) {
+  let targetDocsDir = `${CWD}/../docs`;
+  if (fs.existsSync(targetDocsDir)) {
     console.log(
-      `- ${chalk.green(
-        'docs-examples-from-docusaurus',
-      )} already exists in ${chalk.blue(outerFolder)}.`,
+      `- ${chalk.green('docs')} already exists in ${chalk.blue(
+        outerFolder,
+      )}. Copying into ${CWD}/../docs-examples-from-docusaurus instead.`,
     );
-  } else {
-    fs.copySync(
-      `${folder}/docs-examples-from-docusaurus`,
-      `${CWD}/../docs-examples-from-docusaurus`,
-    );
-    exampleSiteCreated = true;
-    docsCreated = true;
+    targetDocsDir = `${CWD}/../docs-examples-from-docusaurus`;
+    docsCreatedInIntendedDirectory = false;
   }
+
+  fs.copySync(`${folder}/docs`, targetDocsDir);
+
   // copy blog examples
-  if (fs.existsSync(`${CWD}/blog-examples-from-docusaurus`)) {
+  if (fs.existsSync(`${CWD}/blog`)) {
     console.log(
-      `- ${chalk.green(
-        'blog-examples-from-docusaurus',
-      )} already exists in ${chalk.blue(`${outerFolder}/website`)}.`,
+      `- ${chalk.green('blog')} already exists in ${chalk.blue(
+        `${outerFolder}/website`,
+      )}.`,
     );
   } else {
-    fs.copySync(
-      path.join(folder, 'blog-examples-from-docusaurus'),
-      path.join(CWD, 'blog-examples-from-docusaurus'),
-    );
-    exampleSiteCreated = true;
-    blogCreated = true;
+    fs.copySync(path.join(folder, 'blog'), path.join(CWD, 'blog'));
```

Comment:
We dont copy to blog examples if it doesnt exist ?

Reply:
Most people won't have a `blog` dir so the copy to `blog` step will pass.